docs/running.md

```diff
@@ -667,3 +667,43 @@ $ cargo run -p fendermint_rpc --release --example simplecoin -- --secret-key tes
 ```
 
 Note that the script figures out the Alice's nonce on its own, so we don't have to pass it in. It also has an example of running an EVM view method (which is read-only) either as as a distributed read-transaction (which is included on the chain and costs gas) or a query anwered by our node without involving the blockchain. Both have their uses, depending on our level of trust.
+
+## Deploy IPC child subnet
+
+### Crate genesis from parent
+Fendermint includes a command to automatically create the genesis file for an IPC child subnet according to the information for the subnet available in its parent. Here's an example of the generation of a genesis file for a subnet that has already been bootstrapped in the parent.
+```console
+cargo run -p fendermint_app -- --network=test genesis --genesis-file test-network/genesis.json ipc from-parent --subnet-id <CHILD_SUBNET_ID> -p <PARENT_ENDPOINT> --parent-gateway <PARENT_GATEWAY_CONTRACT> --parent-registry <PARENT_REGISTRY_CONTRACT>
```

Comment:
```suggestion
cargo run -p fendermint_app -- \
  --network=test \
  genesis --genesis-file test-network/genesis.json ipc \
    from-parent \
      --subnet-id <CHILD_SUBNET_ID> \
      -p <PARENT_ENDPOINT> \
      --parent-gateway <PARENT_GATEWAY_CONTRACT> \
      --parent-registry <PARENT_REGISTRY_CONTRACT>
```
So readers don't have to scroll in the box. 

By the way I found out that there is some way to tell `clap` that it should accept the options regardless of where they appear, so instead of `fendermint --network ... genesis --genesis-file ... ipc from-parent ...` we could type `fendermint genesis ipc from-parent --network ... --genesis-file ...`. Not sure which one is easier to get right.